src/Inkluzitron/Modules/BdsmTestOrg/BdsmModule.cs

```diff
@@ -23,30 +27,22 @@ namespace Inkluzitron.Modules.BdsmTestOrg
     [Summary("Dotaz na kategorie může mít následující podoby:\n`dom sub` == `dom>50 sub>50` == `+dom +sub`\n`dom -switch` == `dom switch<50`")]
     public class BdsmModule : ModuleBase
     {
-        static private readonly Regex TestResultRegex = new(
-            @"==\sResults\sfrom\sbdsmtest.org\s==\s+
-    (?<results>.+)
-    (?<link>https?://bdsmtest\.org/r/[\d\w]+)",
-            RegexOptions.IgnorePatternWhitespace | RegexOptions.Singleline
-        );
-
-        static private readonly Regex TestResultItemRegex = new(
-            @"^(?<pctg>\d+)%\s+(?<trait>[^\n]+?)\s*$",
-            RegexOptions.Multiline
-        );
+        static private readonly Regex TestResultLinkRegex = new(@"^https?://bdsmtest\.org/r/([\d\w]+)");
 
         private BotDatabaseContext DbContext { get; set; }
         private DatabaseFactory DatabaseFactory { get; }
         private ReactionSettings ReactionSettings { get; }
         private BdsmTestOrgSettings Settings { get; }
         private GraphPaintingService GraphPainter { get; }
+        private HttpClient HttpClient { get; }
 
         public BdsmModule(DatabaseFactory databaseFactory, ReactionSettings reactionSettings, BdsmTestOrgSettings bdsmTestOrgSettings, GraphPaintingService graphPainter)
         {
             DatabaseFactory = databaseFactory;
             ReactionSettings = reactionSettings;
             Settings = bdsmTestOrgSettings;
             GraphPainter = graphPainter;
+            HttpClient = new();
```

Comment:
Používat HttpClient napřímo se nedoporučuje, protože to cachuje DNS/networking věci a bůhvíco. Chtělo by to přidat do DI kontejneru ```.AddHttpClient()```, tady místo ```new()``` injectnout ```IHttpClientFactory``` a z ní si na požádání (tzn. v podstatě na vykonání commandu) vytvořit short-lived klienta.

https://code-maze.com/using-httpclientfactory-in-asp-net-core-applications/

Reply:
To je na mě moc chytrých slov 😀
Na StackOverflow random týpek psal že HttpClient best

Reply:
1. Přidáš do Program.cs .AddHttpClient() (možná bude potřeba nainstalovat NuGeT balíček Microsoft.Extensions.HttpClient)
2. Tady do parametrů konstruktoru přidáš IHttpClientFactory a uložíš ji do property
3. Tam kde se teď používá ten HttpClient z L45 tak místo toho zavoláš Factory.CreateClient(), a použiješ ho.

Reply:
Je to tam - 82f0e9c0